### PR TITLE
AB#39055 and AB#39077 Added get chat and get chat messages queries

### PR DESCRIPTION
**Pull request by omersayshi**
### Overview

Adding chat queries to application mode.

### Testing Instructions

* Follow this instructions of this PR https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/43 to have the functionality on Graph Explorer
* {ADD_YOUR_BRANCH_NAME_HERE} here would be this branch's name {interns/omersayshi/39055-get-chat-query}

### CODE REVIEW

**Review by gracet9182 — APPROVED** [review]
Nice, ran the branch and saw them

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -2983,6 +2983,25 @@
     }
   ],
   "TeamsAppSampleQueries": [
-      
+    {
+        "id": "eb9b7f19-491b-423f-a821-9975615022a4",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -2983,6 +2983,25 @@
...
+        "method": "GET",
+        "humanName": "chat info",
+        "requestUrl": "/beta/chats/{chat-id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-1.0&tabs=http",
+        "tip": "This query requires a chat id. To find a list of chats and their corresponding ids, run GET https://graph.microsoft.com/beta/me/chats/",
+        "skipTest": false
+    },
+    {
+        "id": "acc823bc-f501-4cce-9441-06cbfa2f2884",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

**Review by ayubun — CHANGES_REQUESTED** [review]
Make sure to use the (beta) tag on the category names! Otherwise, it shows up in a different category ^^;

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -3146,5 +3146,25 @@
         "tip": "This query requires a team id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
         "skipTest": false
     }
+    {
+        "id": "eb9b7f19-491b-423f-a821-9975615022a4",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "chat info",
+        "requestUrl": "/beta/chats/{chat-id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-1.0&tabs=http",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-beta",
```

File: sample-queries/sample-queries.json [hunk 4]
```diff
@@ -3146,5 +3146,25 @@
...
+        "tip": "This query requires a chat id. To find a list of chats and their corresponding ids, run GET https://graph.microsoft.com/beta/me/chats/",
+        "skipTest": false
+    },
+    {
+        "id": "acc823bc-f501-4cce-9441-06cbfa2f2884",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "chat messages",
+        "requestUrl": "/beta/chats/{chat-id}/messages",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-1.0&tabs=http",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-beta",
```

**Pull request merged**